packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -7,17 +7,62 @@
 // --------------------------------------------------
 // Search box styles
 
-
-$SearchBox-field-padding-left: 10px;
-$SearchBox-height: 36px;
-
 .root {
   @include ms-font-m;
   @include ms-u-normalize;
   color: $ms-color-neutralPrimary;
   position: relative;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: stretch;
   margin-bottom: 10px;
   border: 1px solid $ms-color-themeTertiary;
+  height: 32px;
+}
+
+.icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: $ms-color-neutralSecondaryAlt;
+  font-size: 16px;
+  flex-basis: 32px;
+  text-align: right;
+  transition: flex-basis $ms-duration1;
+  color: $ms-color-themePrimary
+}
+
+// Input Field
+.field {
+  @include ms-u-normalize;
+  @include padding(0px, 0, 1px, 8px);
+  border: none;
+  outline: transparent 1px solid;
+  font-weight: inherit;
+  font-family: inherit;
+  font-size: inherit;
+  color: $ms-color-black;
+  background-color: transparent;
+  flex-grow: 1;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  &::-ms-clear {
+    display: none;
```

Comment:
This was added in a previous commit. Edge adds a clear button to inputs by default. But it's not keyboard accessible. So we chose to hide it and make functionality consistent between browsers.